plejd/ble.bluez.js

```diff
@@ -379,8 +377,20 @@ class PlejdService extends EventEmitter {
     this.characteristics.lastData.StartNotify();
   }
 
+  async throttledInit(delay) {
```

Comment:
If multiple errors is thrown before the timeout is done, we do not need to re-init multiple times (which results is even more errors since we teared everything down when trying to re-init while re-initing)